dali/kernels/test/static_switch_test.cc

```diff
@@ -69,3 +70,40 @@ TEST(StaticSwitch, Nested) {
   // check that the test functor was actually called
   EXPECT_EQ(calls, 4) << "Test functor was expected to be called 4 times";
 }
+
+#include "dali/pipeline/data/types.h"  // NOLINT
```

Comment:
Please add a comment why this include cannot be placed together with other one.

Reply:
In another PR maybe?

Reply:
I'm afraid that file won't be touched soon. So I guess now or never.

Reply:
OK, will do.